models/DataObject/ClassDefinition/Data/Geopolyline.php

```diff
@@ -127,7 +127,7 @@ public function getDataForEditmode($data, $object = null, $params = [])
     /**
      * @see Data::getDataFromEditmode
      *
-     * @param DataObject\Data\Geopoint[]|null $data
+     * @param array|null $data
```

Comment:
why ?

Reply:
Should be an multidiminsional array with `longitude` and `latitude` keys

Reply:
right, mixed this up with getDataForEditmode :-)